src/config.h

```diff
@@ -106,22 +106,20 @@ extern enum natt_mode_enum opt_natt_mode;
 extern enum if_mode_enum opt_if_mode;
 extern uint16_t opt_udpencapport;
 
-#define TIMESTAMP() ({              \
```

Comment:
this creates a new scope, which then ultimately yields a pointer to an array that just went out of scope when printf calls it.